components/modal/style/modal.less

```diff
@@ -30,6 +30,7 @@
     font-weight: 500;
     font-size: @font-size-lg;
     line-height: 22px;
+    word-wrap: break-word;
```

Comment:
Why adding this? `break-word` should not be default style.

Reply:
Did I break something?